### ISSUE TITLE
order/order not working with scss properties shorthand

### ISSUE THREAD

**Issue by aweiher** [hudochenkov/stylelint-order#20]
Hi, 

I have the problem that `stylelint-order´ recognize *scss properties shorthand* as a rule and not as a declaration. 

Example: 

```scss
.some-rule {
    display: inline-block;
    padding: {
        top: 3px;
        right: 6px;
        bottom: 3px;
        left: 7px;
    }
    text-transform: uppercase;
}
```

Error: 

```text
 9:5  ✖  Expected declaration to come before rule   order/order                  
 9:5  ✖  Expected empty line before declaration     declaration-empty-line-before
```

My Config for declaration block order: 

```
[
    "custom-properties",
    "dollar-variables",

    // declarations and rules in the middle
    "declarations",
    "rules",

    // @content rule
    {
        type: 'at-rule',
        name: 'content',
    },

    // rules to overwrite depending on theme
    {
        type: 'at-rule',
        name: 'include',
        parameter: 'on-[a-zA-Z]*-theme'
    },

    // mq and print mixins at the bottom
    {
        type: 'at-rule',
        name: 'include',
        parameter: "mq"
    },
    {
        type: 'at-rule',
        name: 'include',
        parameter: "print"
    }
]
```

I think the error is not correct, as `padding` is a *declaration* and not a rule.

I am not sure if the following issues are related, but I will include them here:

- https://github.com/stylelint/stylelint/issues/1386
- https://github.com/postcss/postcss-scss/pull/40
- https://github.com/kristerkari/stylelint-scss/issues/44
- https://github.com/postcss/postcss-selector-parser/issues/64
- https://github.com/stylelint/stylelint/issues/1500

Apart from this, migrating from `scsslint` to`stylelint` is very smooth, good job people :)

**Comment by hudochenkov** [comment]
Thank you for reporting and research! I'll look into it soon.

**Comment by hudochenkov** [comment]
Fixed in `0.4.2`.
